server/server.js

```diff
@@ -1,30 +1,38 @@
 require('babel-register');
 const express = require('express');
-// const bodyparser = require('body-parser');
 const React = require('react');
 const ReactDOMServer = require('react-dom/server');
 const ReactRouter = require('react-router-dom');
 const _ = require('lodash');
 const fs = require('fs');
+// const graphqlHTTP = require('express-graphql');
+// const { GraphQLServer } = require('graphql-yoga');
+const bodyParser = require('body-parser');
+// const gqlSchema = require('./schema.js');
 const App = require('../src/app/app').default;
 
 const { StaticRouter } = ReactRouter;
 const baseTemplate = fs.readFileSync(`${__dirname}/../src/index.html`);
 const template = _.template(baseTemplate); // returns a function
 
 const app = express();
-const db = require('./../database/database.js');
```

Comment:
Do we not need this?

Reply:
I meant to just comment it out, but probably not. With GraphQL we will be querying the database differently. We can add it back later if needed.